go/mysql/flavor.go

```diff
@@ -308,6 +308,7 @@ func parseReplicationStatus(fields map[string]string) ReplicationStatus {
 		// These fields are returned from the underlying DB and cannot be renamed
 		IOThreadRunning:  fields["Slave_IO_Running"] == "Yes" || fields["Slave_IO_Running"] == "Connecting",
```

Comment:
Shouldn't this be changed back to just be true on "Yes"?

Reply:
For that to happen we would have to update internal dependencies on this field that expect both Yes and Connecting to be equivalent to true. Happy to make that change if you think it's the best approach.

Reply:
Made the change